x-pack/test/alerting_api_integration/common/fixtures/plugins/actions_simulators/server/unsecured_actions_simulation.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import {
+  RequestHandlerContext,
+  KibanaRequest,
+  KibanaResponseFactory,
+  IKibanaResponse,
+  IRouter,
+} from '@kbn/core/server';
+import { PluginStartContract as ActionsPluginStartContract } from '@kbn/actions-plugin/server';
+
+export function initPlugin(router: IRouter, actionsStart: ActionsPluginStartContract) {
+  router.post(
+    {
+      path: `/api/sample_unsecured_action`,
+      validate: {
+        body: schema.object({
+          requesterId: schema.string(),
+          id: schema.string(),
+          params: schema.recordOf(schema.string(), schema.any()),
```

Comment:
NIT/optional: add proper definition schema for `params` ( and remove the `<any, any, any, any>` from the `req` parameter of the handler to correctly use payload type inference from schema)

Reply:
The parameters schema will change depending on what the connector type is. Definitely something we can tighten up in the future though!